[program:auth]
command=/root/venvs/mems/bin/python -um nodes.rest.auth
directory=/mems/
autorestart=true
startretries=3
stderr_logfile=/var/log/mems/auth_err.log
stdout_logfile=/var/log/mems/auth_out.log
user=root
